ElevenLabs/Packages/com.rest.elevenlabs/Runtime/TextToSpeech/TextToSpeechEndpoint.cs

```diff
@@ -256,6 +257,323 @@ void StreamCallback(Response partialResponse)
             }
         }
 
+
+
+
+
+        /// <summary>
+        /// Converts text into speech using a voice of your choice and returns audio and a list of timestamped characters.
+        /// </summary>
+        /// <param name="text">
+        /// Text input to synthesize speech for. Maximum 5000 characters.
+        /// </param>
+        /// <param name="voice">
+        /// <see cref="Voice"/> to use.
+        /// </param>
+        /// <param name="voiceSettings">
+        /// Optional, <see cref="VoiceSettings"/> that will override the default settings in <see cref="Voice.Settings"/>.
+        /// </param>
+        /// <param name="model">
+        /// Optional, <see cref="Model"/> to use. Defaults to <see cref="Model.MonoLingualV1"/>.
+        /// </param>
+        /// <param name="outputFormat">
+        /// Output format of the generated audio.<br/>
+        /// Defaults to <see cref="OutputFormat.MP3_44100_128"/>
+        /// </param>
+        /// <param name="optimizeStreamingLatency">
+        /// Optional, You can turn on latency optimizations at some cost of quality.
+        /// The best possible final latency varies by model.<br/>
+        /// Possible values:<br/>
+        /// 0 - default mode (no latency optimizations)<br/>
+        /// 1 - normal latency optimizations (about 50% of possible latency improvement of option 3)<br/>
+        /// 2 - strong latency optimizations (about 75% of possible latency improvement of option 3)<br/>
+        /// 3 - max latency optimizations<br/>
+        /// 4 - max latency optimizations, but also with text normalizer turned off for even more latency savings
+        /// (best latency, but can mispronounce eg numbers and dates).
+        /// </param>
+        /// <param name="cancellationToken">Optional, <see cref="CancellationToken"/>.</param>
+        /// <returns>A tuple containing the <see cref="VoiceClip"/> and an array of <see cref="TimestampedTranscriptCharacter"/>.</returns>
+        public async Task<(VoiceClip voiceClip, TimestampedTranscriptCharacter[] timestampedTranscriptCharacters)> TextToSpeechWithTimestampsAsync(string text, Voice voice, VoiceSettings voiceSettings = null, Model model = null, OutputFormat outputFormat = OutputFormat.MP3_44100_128, int? optimizeStreamingLatency = null, CancellationToken cancellationToken = default)
```

Comment:
for a public API I prefer not to return anon Tuples in this way. I'd prefer to create a class definition for the signature, to make breaking changes easier to deal with in future.